ntbs-service/Pages/Errors/403.cshtml

```diff
@@ -0,0 +1,17 @@
+@page

```

Comment:
Just remembered this was missing (usually we show the unauthorized page but there are cases where we return an ordinary forbid result)